<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QPushButton" name="singleshot_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="automode_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>Udp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tcp</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>202</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>发送模式</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
